e2e/playwright/pagination/src/tests/multiple-pages-files.spec.ts

```diff
@@ -22,29 +22,37 @@
  * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { ApiClientFactory, NodesApi, test, Utils } from '@alfresco/playwright-shared';
+import { ApiClientFactory, FavoritesPageApi, NodesApi, test, Utils } from '@alfresco/playwright-shared';
 import { personalFilesTests } from './personal-files';
+import { favoritesTests } from './favorites';
 
 test.describe('Pagination on multiple pages : ', () => {
   const random = Utils.random();
   const username = `user-${random}`;
 
   const parent = `parent-${random}`;
   let parentId: string;
+  let filesIds: string[];
```

Comment:
you can move it to beforeAll and make it as const as it is used only there

Reply:
moved to beforeAll